tests/testthat/test-mlaps.R

```diff
@@ -0,0 +1,171 @@
+# Helper function to create dummy lab data needed for mLAPS unit tests
+dummy_lab <- function(id, omop, value, unit, mintime){
```

Comment:
Nice! Should we actually add this to `dummy_data.R`? I know this may currently not be useful for other functions, but it might be in the future and is probably easier to find in the same spot as the other `dummy_data` functions.